afs/media/js/views/components/reports/observation.js

```diff
@@ -44,6 +44,32 @@ define([
             self.substanceCards = {};
             self.summary = params.summary;
 
+            self.relatedObjectsTableConfig = {
+                ...self.defaultTableConfig,
+                columns: Array(3).fill(null)
+            };
+
+            self.collectionOfRelatedObjects = ko.observableArray();
+            self.visible = {
+                relatedObjects: ko.observable(true),
+            };
+
+            const resourceId = ko.unwrap(self.reportMetadata).resourceinstanceid;
+            const loadRelatedResources = async() => {
+                const result = await reportUtils.getRelatedResources(resourceId);
+                const physicalThingGraphID = '9519cb4f-b25b-11e9-8c7b-a4d18cec433a';
+                const relatedPhysicalThings = result?.related_resources.filter(resource => 
+                    resource?.graph_id === physicalThingGraphID);
+                
+                self.collectionOfRelatedObjects(relatedPhysicalThings.map(element => {
```

Comment:
The link in the related-object is not working. getResourceLink is not going to work this way.
Also, this may be intentional but you can create a new object rather than adding to itself.